### PR TITLE
feat: add artifact scraper resources

### PR DESCRIPTION
**Pull request by vsukhin**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [x] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: charts/testkube-operator/templates/tests.testkube.io_tests.yaml [hunk 1]
```diff
@@ -438,6 +438,25 @@ spec:
...
+                    artifactRequest:
+                      description: artifact request body for container executors with
+                        test artifacts
+                      properties:
+                        dirs:
+                          description: artifact directories
+                          items:
+                            type: string
+                          type: array
+                        volumeMounthPath:
```
Comment (vLia) [inline charts/testkube-operator/templates/tests.testkube.io_tests.yaml:450]: Same typo here
```suggestion
                        volumeMountPath:
```

File: charts/testkube-operator/templates/tests.testkube.io_tests.yaml [hunk 2]
```diff
@@ -438,6 +438,25 @@ spec:
...
+                          description: artifact volume mount path
+                          type: string
+                        volumeName:
+                          description: artifact volume name
+                          type: string
+                      required:
+                      - volumeMounthPath
```
Comment (vLia) [inline charts/testkube-operator/templates/tests.testkube.io_tests.yaml:457]:
```suggestion
                      - volumeMountPath
```

**Review by vLia — APPROVED** [review]
Looks good, small observation

**Comment by vsukhin** [comment]
Cool,thank you!

**Pull request merged**